src/modules/aavev2/logic.borrow.ts

```diff
@@ -0,0 +1,56 @@
+import { InterestRateMode, LogicOptions, serviceType } from './types';
+import { LendingPool__factory } from './contracts';
+import * as common from '@protocolink/common';
+import * as core from '@protocolink/core';
+
+export type BorrowLogicTokenList = common.Token[];
+
+export type BorrowLogicFields = core.TokenOutFields<{ interestRateMode: InterestRateMode; referralCode?: number }>;
+
+export type BorrowLogicOptions = Pick<core.GlobalOptions, 'account'>;
+
+export abstract class BorrowLogic
+  extends core.Logic
+  implements core.LogicTokenListInterface, core.LogicBuilderInterface
```

Comment:
can we move the `implements ...` back to logics/protocol/logic.xxx.ts as univ3?

Reply:
Sure, does that make more sense to you? The current way only extends the module logic, which keeps the final logic class cleaner.

Reply:
updated